pkg/sdn/plugin/node_iptables.go

```diff
@@ -95,6 +95,7 @@ func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
+		{"filter", "FORWARD", []string{"-s", n.clusterNetworkCIDR, "-m", "conntrack", "--ctstate", "INVALID", "-j", "DROP"}},
```

Comment:
@danwinship Are you referencing the rule I just added in this PR that drops invalid packets? It leaked packets with FIN or RST (sdn subnet). The default rule, immediately below it, is "-A FORWARD -s 10.1.0.0/16 -j ACCEPT". I am not all that familiar with how the iptable rules are set up, could you help me figure this out? See bz1438762